packages/components/src/buttons/src/Button.module.css

```diff
@@ -207,46 +239,122 @@
     --border: var(--hop-Button-upsell-border-disabled);
 }
 
-.hop-Button--upsell[data-hovered] {
+.hop-Button--upsell[data-hovered]:not([data-disabled], [data-loading]) {
     --background-color: var(--hop-Button-upsell-background-color-hover);
     --color: var(--hop-Button-upsell-color-hover);
     --border: var(--hop-Button-upsell-border-hover);
 }
 
-.hop-Button--upsell[data-pressed] {
+.hop-Button--upsell[data-pressed]:not([data-loading]) {
     --background-color: var(--hop-Button-upsell-background-color-pressed);
     --color: var(--hop-Button-upsell-color-pressed);
     --border: var(--hop-Button-upsell-border-pressed);
 }
 
-.hop-Button--negative {
-    --background-color: var(--hop-Button-negative-background-color);
-    --color: var(--hop-Button-negative-color);
-    --border: var(--hop-Button-negative-border);
+.hop-Button--danger {
+    --background-color: var(--hop-Button-danger-background-color);
+    --color: var(--hop-Button-danger-color);
+    --border: var(--hop-Button-danger-border);
+    --spinner-color: var(--hop-Button-danger-spinner-color);
+}
+
+.hop-Button--danger[data-disabled]:not([data-loading]) {
+    --background-color: var(--hop-Button-danger-background-color-disabled);
+    --color: var(--hop-Button-danger-color-disabled);
+    --border: var(--hop-Button-danger-border-disabled);
+}
+
+.hop-Button--danger[data-hovered]:not([data-disabled], [data-loading]) {
+    --background-color: var(--hop-Button-danger-background-color-hover);
+    --color: var(--hop-Button-danger-color-hover);
+    --border: var(--hop-Button-danger-border-hover);
+}
+
+.hop-Button--danger[data-pressed]:not([data-loading]) {
+    --background-color: var(--hop-Button-danger-background-color-pressed);
+    --color: var(--hop-Button-danger-color-pressed);
+    --border: var(--hop-Button-danger-border-pressed);
 }
 
-.hop-Button--negative[data-disabled]:not([data-loading]) {
-    --background-color: var(--hop-Button-negative-background-color-disabled);
-    --color: var(--hop-Button-negative-color-disabled);
-    --border: var(--hop-Button-negative-border-disabled);
+.hop-Button--ghost-primary {
```

Comment:
The ghost button's inline padding must be smaller `--hop-space-inset-sm` than the standard button.

<img width="289" alt="Capture d’écran, le 2024-03-28 à 15 09 06" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/b54b6b07-851b-4084-b2c0-98e901f47a1d">